doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -355,23 +433,26 @@ Geographical queries are specified by means of the `geometry` and `coords` param
 
 **geometry** defines a geografical area so that only those entities located in such area match the query.
 It is composed of a token list separated by ';'. The first token is the shape of the geometry,
-the rest of tokens (if any) depends on the shape. 
-
-In particular, the following geometries are supported:
-
-+ `geometry=circle;radius:4000`, inside a circle (radius is in meters).
-+ `geometry=circle;radius:4000;external`, outside a circle (radius is in meters).
-+ `geometry=polygon`, inside a polygon.
-+ `geometry=polygon;external`, outside a polygon.
-
-**coords** represents a a list of pairs of geographical coordinates (latitude, longitude) separated by ','.
-Every pair of coordinates represent a geographical point. The list length and its semantics depend on the shape
-specified by the **geometry** parameter:
-
-+ In the case of `circle` geometry, the parameter includes just one element, which
-  is the center of the circle.
-+ In the case of `polygon` includes a list of elements, which are the vertices of the polygon.
-
+the rest of tokens (if any) depends on the shape. The following geometries (see Simple Location Format)
+must be supported:
+
++ `geometry=point`, entities located at exactly that point on the Earth surface. 
++ `geometry=circle;radius:4000`, entities inside a circle of the given radius (radius unit must be meters).
++ `geometry=circle;radius:4000;external`, entities outside a circle of the given radius (radius unit must be meters).
++ `geometry=polygon`, entities inside a polygon.
++ `geometry=polygon;external`, entities outside a polygon.
++ `geometry=box`, entities inside a bounding box.
++ `geometry=box;external`, entities outside a bounding box. 
+
+**coords** represents a a list of geographical coordinates as mandated by the Simple Location Format
+and according to the geometry specified.
+In the particular case of the `circle` geometry the `coords` parameter must include just one coordinate pair,
+which is the center of the circle.
+
+*Implementations are responsible for determining the entity attributes which convey geospatial properties*. Typically
```

Comment:
Yes of course, geospatial properties will be conveyed by an entity attribute. But I don't think we need metadata for that purpose. Actually metadata will be used for stating other important parameters such as accuracy. The NGSI type itself can be used to mark the attribute (geo:point, etc.) or GeoJSON  and to recognize that it refers to a geographical coordinate. 

But I think there can be more than one attribute that express a GeoLocation and in that particular case for query resolution it might be needed to indicate what attribute must be taken for resolving geo-queries. 

>> Another ongoing issue is if metadata properties of attributes might contain as well geospatial data.

What I was referring here is to the following use case:

```json
{
   id: "C1234",
   name: {
     value: "Madrid"
   } ,
   temperature: {
      value: 34,
      metadata: {
         location: {
            type: "geo:point",
            value: "44.5, 2.6"
         }
      }
  }
}
```
Is that something we want to support? The original OMA NGSI spec talks about this possibility, but I don't think it is relevant because the same info can be captured using other entities and a regular attribute. What do you think?